src/App.tsx

```diff
@@ -1,56 +1,89 @@
-import React from 'react';
 import './App.scss';
+import React, { useCallback, useMemo, useState } from 'react';
+import cn from 'classnames';
+import { Person } from './types/Person';
 import { peopleFromServer } from './data/people';
+import { Dropdown } from './components/DropdownList';
+import { debounce } from './debounce';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [people] = useState(peopleFromServer);
+  const [searchQuery, setSearchQuery] = useState('');
+  const [appliedQuery, setAppliedQuery] = useState('');
+  const [isSuggestions, setIsSuggestions] = useState(false);
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>();
+
+  const applyQuery = useCallback(
+    debounce(setAppliedQuery, 1000),
+    [],
+  );
+
+  const { name, born, died } = selectedPerson ?? {};
+
+  const preparedSearchQuery = searchQuery.toLowerCase().trim();
+
+  const visiblePeople = useMemo(() => {
+    return people.filter(
+      person => person.name.toLowerCase().includes(preparedSearchQuery),
+    );
+  }, [appliedQuery, people]);
```

Comment:
if you don't have any actions before the return statement - you can return right away, using this syntax:  `=> (...here's what you return)`